doc/doxygen/references.bib

```diff
@@ -1217,7 +1217,7 @@ @book{Korelc2016a
 
 @inbook{Truesdell1960a,
   author    = {C. Truesdell and R. Toupin},
-  title     = {Encyclopedia of Physics: Principles of Thermodynamics and Statics},
+  title     = {Encyclopedia of Physics: Principles of Thermodynamics and Statistics},
```

Comment:
The title was wrong anyways. 
Following the DOI link I can see that the sub-series name is 
'Principles of Theoretical Physics' 
The 'Principles of Thermodynamics and Statistics' was the previous book in the series.

Reply:
I've updated the entry. Can you have another look?

Reply:
Looks good now!